pyscriptjs/tests/integration/test_async.py

```diff
@@ -87,7 +89,7 @@ async def b_func():
             "b func done",
         ]
 
-    @skip_worker("FIXME: display()")
+    @pytest.mark.skip("FIXME: display() without target is broken")
```

Comment:
once again, `display` is not broken ... here the test working just fine using pyscript next

```html
<!doctype html>
<script type="module" src="https://cdn.jsdelivr.net/npm/@pyscript/core"></script>
<py-script id='pyA'>
  import asyncio
  from pyscript import display

  async def a_func():
      for i in range(2):
          display(f'A{i}', append=True)
          await asyncio.sleep(0.1)

  asyncio.ensure_future(a_func())
</py-script>
<py-script id='pyB'>
  import asyncio
  from pyscript import display, window

  async def a_func():
      for i in range(2):
          display(f'B{i}', append=True)
          await asyncio.sleep(0.1)
      window.console.log("B DONE")

  asyncio.ensure_future(a_func())
</py-script>
```

Reply:
I've tested a few of the `display` cases on chrome yesterday and observed the same issue. Maybe #1658 may have fixed it? 🤷 

Either way, if it's a false negative even better.

Reply:
tests here don't import `display` this has nothing to do with #1658 ... all code works down to previous releases. The tests re all based on global leaks and code that is not PyScript next and it doesn't use PyScript next features.

If you don't import `display` you are not using PyScript next.